chrome/fed-modules.json

```diff
@@ -379,7 +379,7 @@
                 "id": "ansible-automation-analytics",
                 "module": "./RootApp",
                 "routes": [
-                    "/ansible/insights",
+                    "/ansible/automation-analytics",
                     "/ansible/reports",
```

Comment:
What about the rest of the routes on the list? Is the automation analytics app supposed to render there? I would guess no based on the links from the nav.

Reply:
OK lest take clean them up in another PR